installer/knative/wait-for-ingress.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+kubectl wait --for=condition=available --timeout=600s deployment -l networking.knative.dev/ingress-provider=istio -n knative-serving
```

Comment:
This is not really waiting for `ingress`, I would name this file something like `wait-for-config.sh`

Reply:
ok